editing/edit-features-with-feature-linked-annotation/src/main/java/com.esri.samples.edit_features_with_feature_linked_annotation/EditAttributesDialog.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.edit_features_with_feature_linked_annotation;
+
+import java.util.concurrent.ExecutionException;
+
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+import javafx.scene.control.TextField;
+
+import com.esri.arcgisruntime.data.Feature;
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+
+public class EditAttributesDialog extends Dialog<String>{
```

Comment:
Let's add a short description to this class, just for some context. An example based on some other classes extending dialog:
```
/**
 * Custom dialog for editing feature attributes.
 */
```

Also, although it has no direct implications in the way we are using it, it would be more correct to define this as:

```suggestion
public class EditAttributesDialog extends Dialog<Feature>{
```

Since we are using this dialog to interact with a particular feature (as you have it in the constructor).